landscape.yml

```diff
@@ -8034,6 +8072,20 @@ landscape:
             logo: flant-deckhouse.svg
             twitter: https://twitter.com/deckhouseio
             crunchbase: https://www.crunchbase.com/organization/flant
+          - item:
+            name: Flatcar Container Linux
+            description: >-
+              A community Linux distribution designed for container workloads, with high security and low maintenance
+            homepage_url: https://www.flatcar.org/
+            repo_url: https://github.com/flatcar/Flatcar
+            logo: flatcar.svg
+            project: incubating
+            twitter: https://twitter.com/flatcar
```

Comment:
Don't you need a `second_path` entry here to point to the category you're removing it from below?

Reply:
weird, I thought I had - but guess I missed it - will add it now quick

Reply:
Updated :+1: